File: SportAppServer/SportAppServer/SportAppServer/Models/DTOs/CommentDTO.cs
using System.Text.Json.Serialization;
using SportAppServer.Models.DTOs;

namespace SportAppServer.Models.Entities
{
    public class CommentDTO
    {
        [JsonPropertyName("comment_id")]
        public int CommentId { get; set; }

        [JsonPropertyName("news_date_time")]
        public DateTime NewsDateTime { get; set; }

        [JsonPropertyName("comment_date_time")]
        public DateTime CommentDateTime { get; set; }

        [JsonPropertyName("comment_text")]
        public string CommentText { get; set; }

        [JsonPropertyName("email")]
        public string UserEmail { get; set; }

        [JsonPropertyName("user")]
        public UserDTO User { get; set; }

        [JsonPropertyName("elapsed_ime")]
        public string ElapsedTime { get; set; }

        public CommentDTO() { }

        public CommentDTO(int commentId, DateTime newsDateTime, DateTime commentDateTime, string commentText, string userEmail, UserDTO user)
        {
            CommentId = commentId;
            NewsDateTime = newsDateTime;
            CommentDateTime = commentDateTime;
            CommentText = commentText;
            UserEmail = userEmail;
            User = user;
        }

        public override string ToString()
        {
            return $"CommentId: {CommentId}, NewsDateTime: {NewsDateTime}, CommentDateTime: {CommentDateTime}, CommentText: {CommentText}, UserEmail: {UserEmail}";
        }
    }
}

File: app/src/main/java/com/example/sportapp/data/dto/comments/CommentDto.kt
package com.example.sportapp.CleanArchitexture.data.dto.comments

import com.example.sportapp.CleanArchitexture.data.dto.user.UserDto
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class CommentDto (
    @SerialName("comment_id") val commentId: Int,
    @SerialName("news_date_time") val newsDateTime: String,
    @SerialName("comment_date_time") val commentDateTime: String,
    @SerialName("comment_text") val commentText: String,
    @SerialName("email") val email: String,
    @SerialName("user") val user: UserDto,
    @SerialName("elapsed_time") val ElapsedTime: String,
    @SerialName("is_liked") val isLiked: Boolean,
    @SerialName("likes_count") val LikesCount: Int
)
